grammar.js

```diff
@@ -341,6 +342,7 @@ module.exports = grammar({
       ),
     todo: ($) =>
       seq("todo", optional(seq("(", field("message", $.string), ")"))),
+		panic: (_$) => "panic",
```

Comment:
GitHub is showing an indentation mismatch; when the project was started, I arbitrarily decided on two spaces for indentation, but this line has tab characters.

Running `npm run format` will automatically fix that (and other such things) :grin: